src/pages/loginPage.ts

```diff
@@ -0,0 +1,35 @@
+import { Page } from '@playwright/test';
+import PlaywrightWrapper from '../helper/wrapper/playwrightWrappers';
+import basePage from './basePage';
+
+export default class LoginPage {
+  private base: PlaywrightWrapper;
+
+  constructor(private page: Page) {
+    this.base = new PlaywrightWrapper(page);
+  }
+
+  logInBtn = this.page.locator("//a/span[text()='Log in']");
+  emailInput = this.page.locator("//input[@name='email']");
+  passwordInput = this.page.locator("//input[@name='password']");
+  submitBtn = this.page.locator("//button[@type='submit']");
+
+  async logIn(userRole: string) {
+    let email: string;
+    let password: string;
+
+    email = process.env[`${userRole.toUpperCase()}_EMAIL`];
+    password = process.env[`${userRole.toUpperCase()}_PASSWORD`];
```

Comment:
You can instantly set values to variables when you declare them. No need to declare a variable with `undefined` value, and set a value on the following line.